crates/iota-core/src/consensus_manager/starfish_manager.rs

```diff
@@ -128,13 +128,11 @@ impl ConsensusManagerTrait for StarfishManager {
             return;
         };
 
-        let consensus_config = config
-            .consensus_config()
-            .expect("consensus_config should exist");
-
+        // TODO: fill in real values for the rest of the parameters before deploying on
```

Comment:
I recommend to always create tracking issues for such TODOs, and mention them in the code. Otherwise we might forget about those before deployment.